core-ui/src/components/ApiRules/useGetGatewayDomain/useGetGatewayDomain.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useGet, useMicrofrontendContext } from 'react-shared';
+
+const apiGatewayUri =
+  '/apis/networking.istio.io/v1beta1/namespaces/kyma-system/gateways/kyma-gateway';
+
+export function useGetGatewayDomain() {
+  const k8sApiUrl = useMicrofrontendContext().k8sApiUrl;
```

Comment:
do we need to additionally make the call for the gatewayURI later on if we already have the domain?

Reply:
We make no call if the domain is already here - `skip: !!domain,`

Reply:
right :)